superset-frontend/src/explore/controls.jsx

```diff
@@ -197,6 +197,7 @@ export const controls = {
     label: t('Dataset'),
     default: null,
     description: null,
+    healthCheckMessage: null,
```

Comment:
I believe most if not all control configs in this file are not in use any more. We should clean them up.

Reply:
yes. this line has no use :(